formats/bibtex/index.js

```diff
@@ -0,0 +1,22 @@
+import Format from "../../src/format.js";
+import { BibLatexParser, BibLatexExporter } from "https://esm.sh/biblatex-csl-converter/lib/index.js";
+
+export default class BibTeX extends Format {
+	static defaultOptions = {
+		processUnexpected: true,
+		processUnknown: true,
+	};
+
+	static extensions = ["bib"];
```

Comment:
That’s the most popular one, but pretty sure there are a few more extensions in use.